build.gradle

```diff
@@ -16,7 +14,7 @@ buildscript {
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:4.1.2'
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:_"
```

Comment:
@aitorvs I think this is the way we can avoid having to duplicate the kotlin version in 2 files, but not 100% sure. 

By executing `./gradlew buildEnvironment` seems this will work: `org.jetbrains.kotlin:kotlin-gradle-plugin:{require 1.5.20; reject _} -> 1.5.20`, so I guess is ok

Reply:
That's interesting! I seems it works because the `_` is not really a valid version and then it falls back to any other (latest) version we might have. I think a more secure approach might be the following, in the root `build.gradle`

```groovy
buildscript {
    def versionPropsFile = file('versions.properties')
    Properties versionProps = new Properties()

    if (versionPropsFile.canRead()) {
        versionProps.load(new FileInputStream(versionPropsFile))
    }

    ext.kotlin_version = versionProps['version.kotlin']
    ...
```